scripts/duplicate_detector.py

```
@@ -0,0 +1,275 @@
+
+    Returns:
+        list. A list of issue dictionaries.
+    """
+    issues = []
+    page = 1
+    while True:
+        url = (
+            f'https://api.github.com/repos/{repo}/issues?'
+            f'state=open&per_page=100&page={page}'
+        )
+        req = urllib.request.Request(url, headers=headers)
```

Ruff (0.15.20)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)